numba/cgutils.py

```diff
@@ -932,18 +937,16 @@ def memcpy(builder, dst, src, count):
     * dst.type == src.type
     * count is positive
     """
+    # Note this does seem to be optimized as a raw memcpy() by LLVM
+    # whenever possible...
     assert dst.type == src.type
-    with for_range(builder, count, count.type) as loop:
+    with for_range(builder, count, intp=count.type) as loop:
         out_ptr = builder.gep(dst, [loop.index])
         in_ptr = builder.gep(src, [loop.index])
         builder.store(builder.load(in_ptr), out_ptr)
 
 
-def memmove(builder, dst, src, count, itemsize, align=1):
-    """
-    Emit a memmove() call for `count` items of size `itemsize`
-    from `src` to `dest`.
-    """
+def _raw_memcpy(builder, func_name, dst, src, count, itemsize, align):
```

Comment:
does the body of this function need updating to use `func_name`? https://github.com/pitrou/numba/blob/setobj/numba/cgutils.py#L953 suggests `llvm.memmove` is still in use?